maven/impl-maven-embedded/src/main/java/org/jboss/shrinkwrap/resolver/impl/maven/embedded/FileExtractor.java

```diff
@@ -85,8 +85,9 @@ private void extractFileInDestinationDir() {
       } catch (IOException e) {
          System.err.println("Failed to unzip file: " + e.getMessage());
       }
-      markerFileHandler.deleteMarkerFile();
-      System.out.printf("Resolver: Successfully extracted maven binaries from %s%n", fileToExtract);
+      if (markerFileHandler.deleteMarkerFile()) {
```

Comment:
Even though this is just a simple print, I wouldn't make printing the line dependent on deleting the file. Especially if the file is not related to the actual extraction

Reply:
This is because I was trying to fix flaky test that relies on this message.
Failing to delete marker file indicates a failure in the code, at least that's how I am reading this,
so actual functionality should be correct.

Reply:
Got it, found the test. Must admit, not my favorite way of checking whether the file was extracted, but I can understand this change.

Can I ask you to put this in a separate commit? As this isn't related to the project-local dependencies I'd prefer git blame to show a related commit when it comes to checking history of this change

Reply:
yes